pajbot/modules/emote_limit.py

```diff
@@ -20,7 +20,7 @@ class EmoteLimitModule(BaseModule):
             required=True,
             placeholder="",
             default=15,
-            constraints={"min_value": 1, "max_value": 40},
+            constraints={"min_value": 1, "max_value": 167},
```

Comment:
Is there even a reason to have a maximum value?

Reply:
maybe not, but this seems fine